input/pages/capabilities.md

```diff
@@ -34,6 +34,7 @@ In addition, the following example capability statement illustrates the use of t
 ## Operations
 
 * [**Library/$package**](OperationDefinition-Library-package.html)
+* [**Library/$draft**](OperationDefinition-Library-draft.html)
 * [**Library/$data-requirements**](OperationDefinition-Library-data-requirements.html)
```

Comment:
Do these need to be updated?

Reply:
Yes, but the pattern needs to be decided by the CRMI workgroup, so it's undecided and out of scope.